api/client/event/event_stream.go

```diff
@@ -106,7 +106,10 @@ func (h *EventStream) Subscribe(eventsChannel chan<- *Event) {
 	var eventType, data string // Variables to store event type and data
 
 	// Iterate over lines of the event stream
-	for scanner.Scan() {
+	for {
+		if ok := scanner.Scan(); !ok {
+			break
+		}
```

Comment:
Is this change necessary?

Reply:
i don't think so anymore I think I'll revert this